doc/iterations/2024.5/shapes/guarantee_representations_models_compatibility_between_versions.adoc

```diff
@@ -0,0 +1,39 @@
+= Add the support for representations migration
+
+== Context
+
+Users want to upgrade the version of Sirius Web without losing the representatons they created with an older version.
+
+== Key Result
+
+When a user creates a representation with a version n of Sirius Web, the representation created must be available with any version>n of Sirius Web. 
+
+The user must also be able to contribute on how to handle evolutions on his own representation to guarantee compatibility between several versions.
```

Comment:
The user will not do that, contributors or application extending Sirius Web ok but not the end user.